deployment/keystone/changeset/update_nodes.go

```diff
@@ -14,14 +15,31 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/p2pkey"
 )
 
+type MCMSConfig struct {
```

Comment:
new optional MCMS configuration inspired by ccip team

perhaps this should go in common/changeset

Reply:
yes  👍 , i like the idea since the MCMS config is common across different changesets.